### ISSUE TITLE
confirm if sprites are still invisible in Unity Doom

### ISSUE THREAD

**Issue by UndeadRyker** [UndeadRyker/CABIN#4]
I don't remember if I fixed this or not.
See Doomworld Thread: [https://www.doomworld.com/forum/topic/110646-the-unity-port-thread-ps4xbox-oneswitchiosandroid/?page=41&tab=comments#comment-2185050](https://www.doomworld.com/forum/topic/110646-the-unity-port-thread-ps4xbox-oneswitchiosandroid/?page=41&tab=comments#comment-2185050)

**Comment by UndeadRyker** [comment]
Unity fucking crashes on loading the map, but demos for the map load okay, and the sprites are ignored from DEHACKED. Great.

**Comment by UndeadRyker** [comment]
After doing some thinking, at one point CABIN used to be able to at the very least, be played even though the sprites weren't properly replaced. Unfortunately, I wish I kept the WIP update I had in 2021 before I wiped my hard drive, but I thought it wasn't worthy enough to be kept back then for some reason so I'm not sure if I found a workaround for Unity. If my memory isn't lying to me, I thought I somehow figured out how to get Unity to recognize the sprites without having to refactor some things.

However, I can confirm that the map crashing upon selecting a new game was NEVER an issue before. Even if I fix the sprites, the maps won't load for some reason, so it's not worth fixing. Disappointing to have to put a wontfix tag on this.
